doc/crds/daemonset-install.yaml

```diff
@@ -106,7 +104,7 @@ spec:
           - >
             SLEEP=false /install-cni.sh &&
             /ip-control-loop -log-level debug
-        image: ghcr.io/k8snetworkplumbingwg/whereabouts:latest-amd64
+        image: ghcr.io/k8snetworkplumbingwg/whereabouts:latest
```

Comment:
The core intent of the change can be boiled down to this line. Allow installing whereabouts from the example without having to do any modification work based on system architecture.